app/src/main/res/values/strings.xml

```diff
@@ -94,4 +94,5 @@
     <string name="accept">Accept</string>
     <string name="reject">Reject</string>
     <string name="select_all">Select all</string>
+    <string name="see_more">See More...</string>
```

Comment:
WARNING: [TypographyEllipsis] Replace "..." with ellipsis character (…, &#8230;) ?
 You can replace the string "..." with a dedicated ellipsis character, ellipsis character (…, &#8230;). This can help make the text more readable.